vnext/docs/NativeModulesSetup.md

```diff
@@ -124,6 +124,8 @@ After you've added the reference, you need to make sure it doesn't copy itself i
 1. Select `Properties`.
 1. Under `Build`, Change `Copy Local` to `False`.
 
+Also make sure your library project is targeting the same version of C++/WinRT as Microsoft.ReactNative. You can check the version by opening `Microsoft.ReactNative.vcxproj` file located at `node_modules\react-native-windows\Microsoft.ReactNative`. Update the versions in `MyLibrary.vcxproj` if necessary.
```

Comment:
>sure [](start = 10, length = 4)

I made a similar change yesterday to this document, this is what I added:
"For C++/WinRT project, right click on the project and choose `Manage Nuget Packages...`, select version 2.0.190730.2 for Microsoft.Windows.CppWinRT pacakge."

Reply:
Oh I see, I'll remove this part then.

Reply:
Removed